README.md

```diff
@@ -1 +1,40 @@
-# snips-react
\ No newline at end of file
+# snips-react
+
+This VS Code extension adds snippets for React in your editor for typescript.
+
+### TypeScript React Snippets (.tsx)
+
+| Snippet                      | Purpose                                     |
+|------------------------------|---------------------------------------------|
+| `react`                      | Snippet for default import (React & Radium) |
+| `store`                      | Snippet for store.dispatch                  |
+| `saga`                       | Snippet for Saga (watcher & worker)         |
+| `arrow`                      | Snippet for Arrow Function                  |
+| `it`                         | Snippet for it (testing)                    |
+| `unroll`                     | Snippet for unroll (testing)                |
+| `describe`                   | Snippet for describe (testing)              |
```

Comment:
Change the description to `Snippet for the Jest test suite.`

Reply:
Done.